poetry.lock

```diff
@@ -320,11 +320,11 @@ python-versions = "*"
 
 [[package]]
 name = "pathspec"
-version = "0.9.0"
+version = "0.10.1"
 description = "Utility library for gitignore style pattern matching of file paths."
 category = "dev"
 optional = false
-python-versions = "!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*,>=2.7"
+python-versions = ">=3.7"
```

Comment:
Nit: The [lint workflow](https://github.com/transitmatters/new-train-tracker/blob/master/.github/workflows/lint.yml#L14) is using Python 3.8. Should these two match?

Reply:
I would agree that we should pin to at minimum 3.8, no reason to run on anything lower